css/admin.upgrade.css

```
@@ -63,6 +70,7 @@
 	background-color: var(--color-white);
 	border-radius: 3px;
 }
+
 .genesis-custom-blocks-pro .btn {
```

```suggestion
.genesis-custom-blocks-pro .container .btn {
```

This will help fix the styling of the two lower buttons.